cobra/flux_analysis/geometric.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf8 -*-
+
+"Contains functions to perform geometric FBA."
+
+from __future__ import absolute_import
+
+from optlang.symbolics import Zero
+
+from cobra.flux_analysis.parsimonious import add_pfba
+from cobra.flux_analysis.variability import flux_variability_analysis
+from cobra.core import get_solution
+
+
+def geometric_fba(model, epsilon=1E-06):
+    """Perform geometric FBA to obtain a unique
+    centered flux.
```

Comment:
This summary must be one line exactly. It's fine to not start directly after the quotes, however, if size is a problem. I suggest the following change:
```python
"""
Perform geometric FBA to obtain a unique, centered flux distribution.
```

Reply:
Size won't be a problem, I can fit it after the quotes.